source/plot/LinePlotData2D.ooc

```diff
@@ -12,30 +12,42 @@ LineStyle: enum {
 
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
+	dataSeries: VectorList<FloatPoint2D> { get set }
```

Comment:
There is nothing special in that class yet (but stuff like `minValues` etc. further down would fit well there). Let's leave it for now unless @davidhesselbom objects.